### PR TITLE
Add homework-14 solution

### PR DESCRIPTION
**Pull request by BilkoYevgen**
Check it please!

### CODE REVIEW

File: HW-13/main.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            command = ''
+            continue
+
+    # ------------------------------------------------------------------------------------------
+
+    if command == 'add':
+        name = input('Type name: ')
+        number = input('Type number: ')
+        if name not in phonebook:
+            if re.search(r'(\+38|38)?0\d{9}\b', number):      # REGEX IS HERE
```
Comment (klymenok) [inline HW-13/main.py]: Some bad formats can be still validated by this regex. Here are some cases:
```
a0951234567 -> 0951234567
8+30951234567 -> 0951234567
a+380951234567 -> +380951234567
80951234567 -> 0951234567
+830965254141 -> 0965254141

File: HW-13/main.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+def replace_emails(file_name):
+    with open(file_name, "r") as file:
+        text = file.read()
+    email_regex = r"[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+"
+    emails = re.findall(email_regex, text)
+    for email in emails:
+        first_char = email[0]
+        last_char = email[-1]
+        middle = "*" * (len(email) - 2)
```
Comment (klymenok) [inline HW-13/main.py:135]: Too many `*` in the replaces email. This is not a mistake according to the task, but you did it wrong if you wanted to replace the email address with the amount of `*` that equals email's symbols count

**Review by klymenok — CHANGES_REQUESTED** [review]

File: HW-13/main.py (hunk 1 above)
Comment (BilkoYevgen) [reply HW-13/main.py]: Problem solved

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**